sources/console.c

```diff
@@ -649,8 +649,11 @@ static inline int od_console_show_pools_add_cb(od_route_t *route, void **argv)
 					route->stats.query_hgram[i]);
 				td_merge(queries_hgram, freeze_hgram);
 			}
-			td_merge(common_transactions_hgram, transactions_hgram);
-			td_merge(common_queries_hgram, queries_hgram);
+			if (strcmp(route->id.database, "postgres") != 0 &&
+			    strcmp(route->id.database, "pgbouncer") != 0) {
```

Comment:
user does not always name consiole 'pgbouncer'

Reply:
You're right. I guess we can configure a list of databases for skip via the .conf file.